debian/control

```diff
@@ -1,21 +1,35 @@
 Source: aeneas
 Section: python
-X-Python-Version: >= 2.7
 Priority: optional
 Maintainer: Alberto Pettarin <alberto@readbeyond.it>
 Build-Depends: debhelper (>= 9.0.0),
                dh-python,
                python-all-dev,
                python-setuptools,
                python-numpy, python-lxml, python-bs4,
+               python3-all-dev,
+               python3-setuptools,
+               python3-numpy, python3-lxml, python3-bs4,
                libasound2-dev, libsndfile1-dev, libespeak-dev
-Standards-Version: 3.9.5
+Standards-Version: 4.1.4
 Homepage: https://github.com/readbeyond/aeneas
 
 Package: python-aeneas
 Architecture: any
 Depends: ${misc:Depends}, ${python:Depends}, 
          espeak, espeak-data, libespeak1, vorbis-tools, ffmpeg, flac
+Conflicts: python3-aeneas
```

Comment:
Why should they be conflicting? You could have them installed under python2 and python3?

Reply:
I could be wrong in the details, but I think the shell script in /usr/bin retains the same name regardless of the version of python it calls, but it calls a specific version of python.  My effort was to make the minimal change necessary to get python3-aeneas packages working.
I'm not sure why anyone would need both packages installed simultaneously.
Some of what is going on in python package builds is still magic to me, and I was relieved that it worked as well and as easily as it did even if the resulting packages had a conflict in one or two files.

Reply:
changed Conflicts: to Replaces: for both binary packages